src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -326,7 +326,7 @@
 
 helps['resource invoke-action'] = """
     type: command
-    short-summary: Invoke an action on the resource. Must provide either the --action parameter or the --post-url parameter
+    short-summary: Invoke an action on the resource. Must provide the --action parameter
```

Comment:
> Must provide the --action parameter 

In that case the parameter should be [Required] and you don't need to state that here.